transport/grpc/inbound.go

```diff
@@ -58,21 +58,19 @@ func (i *inbound) Start(h transport.Handler, d transport.Deps) error {
 	}
 
 	var serviceDescs []grpc.ServiceDesc
-
-	// TODO generate serviceDescs from yarpc registration info
-	serviceDescs = append(serviceDescs, grpc.ServiceDesc{
-		ServiceName: url.QueryEscape("yarpc"),
-		HandlerType: (*passThroughService)(nil),
-		Methods: []grpc.MethodDesc{
-			{
-				MethodName: url.QueryEscape("yarpc"), // TODO: Is this what we want here?
-				Handler:    gHandler.Handle,          // grpc.methodHandler
-			},
-		},
-		Streams: []grpc.StreamDesc{},
-	})
-
-	// TODO Generate the serviceDesc from the configuration of the dispatcher name and procedure names
+	if reg, ok := h.(transport.Registry); ok {
```

Comment:
Move building serviceDescs to a function and explain in a comment what we're doing here since this is sort-of a hack